site/src/components/JsonSchemaPreview.tsx

```diff
@@ -128,6 +129,37 @@ function Schema({ schema: baseSchema, defs, parent, foldable }: SchemaProps) {
     );
   }
 
+  // Intersection
+  if (schema.anyOf) {
+    const { anyOf, ...common } = schema;
+    let inner = (
+      <ul>
+        {schema.anyOf.map((item: JSONSchema7, index) => {
+          return (
+            <li key={index} className='mt-4 pl-4'>
+              <TypeLabel type={`Variant #${index + 1}`} />
+              <Schema parent={parent} schema={item} defs={defs} foldable={false} />
+            </li>
+          );
+        })}
+      </ul>
+    );
+
+    return (
+      <>
+        <Schema parent={parent} schema={common} defs={defs} foldable>
+          {isFoldable ? (
+            <Foldable title='Show possible variants' closeTitle='Hide possible variants'>
+              {inner}
+            </Foldable>
+          ) : (
+            inner
+          )}
+        </Schema>
+      </>
+    );
```

Comment:
This would result in `<Schema><Schema>{children}</Schema></Schema>`, it should do the same thing as the `oneOf` branch